README.rst

```diff
@@ -80,6 +80,18 @@ How?
   ...     break
   '<!doctype html>\n'
 
+  >>> # to download a whole "directory". Note that object storage doesn't have
+  >>> # real directories, see here for details:
+  >>> # https://stackoverflow.com/questions/38416598/how-to-create-an-empty-folder-on-google-storage-with-google-api/38417397#38417397
+  >>> client = storage.Client()
+  ... bucket = client.get_bucket("my-bucket")
```

Comment:
This isn't valid REPL (the syntax is incorrect). You can verify this by running:

```
python -m doctest README.rst
```

Please do:

1. Move this to howto.rst (it's a bit too edge-casey for the README)
2. Use a publicly accessible bucket and folder (so that you don't need credentials to access it)
3. Fix the syntax ;)
4. Run `python -m doctest howtos.rst` to ensure that everything still works

Thanks!

Reply:
I moved this to the howto docs, which appear to be in Markdown, not Restructured Text, and fixed the syntax, and made it so it's self contained and runs on its own. 

I get a lot of failures in the doctests for the howtos though. Most are because I have no boto credentials, but there were several syntax errors in other tests. Not sure if I did something wrong or that's just how they were.

Reply:
You may ignore the boto tests, we have our own credentials for running those.